Index: closure/goog/ui/controlrenderer.js
===================================================================
--- closure/goog/ui/controlrenderer.js	(revision 1186)
+++ closure/goog/ui/controlrenderer.js	(working copy)
@@ -588,7 +588,7 @@
  * in IE6 and below. See {@link IE6_CLASS_COMBINATIONS} for more detail. This
  * method doesn't reference {@link IE6_CLASS_COMBINATIONS} so that it can be
  * compiled out, but subclasses should return their IE6_CLASS_COMBINATIONS
- * static contasnt instead.
+ * static constant instead.
  * @return {Array.<Array.<string>>} Array of class name combinations.
  */
 goog.ui.ControlRenderer.prototype.getIe6ClassCombinations = function() {
